### PR TITLE
Added a scraping option to set the maximum number of records to retrieve.

### PR DESCRIPTION
**Pull request by mcpierce**
## Status
**READY**

## Migrations
NO

## Description
Added a maximum records option for when fetching the volumes while scraping comics.

### CODE REVIEW

**Review by BRUCELLA2 — APPROVED** [review]

**Comment by BRUCELLA2** [comment]
PR approved but there is some conflicts after the last merge.

**Comment by mcpierce** [comment]
> PR approved but there is some conflicts after the last merge.

kk. i just rebased it but didn't see a conflict show up.

**Pull request merged**
